src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -91,25 +90,18 @@ private void createGeneratorControlEquations(LfBus bus,
 
     private void createLocalVoltageControlEquation(LfBus bus,
                                                    EquationSystem<AcVariableType, AcEquationType> equationSystem) {
-        EquationTerm<AcVariableType, AcEquationType> vTerm = equationSystem.getVariable(bus.getNum(), AcVariableType.BUS_V)
-                .createTerm();
-        bus.setCalculatedV(vTerm);
         if (bus.hasGeneratorsWithSlope()) {
             // take first generator with slope: network loading ensures that there's only one generator with slope
             double slope = bus.getGeneratorsControllingVoltageWithSlope().get(0).getSlope();
 
             // we only support one generator controlling voltage with a non zero slope at a bus.
             // equation is: V + slope * qSVC = targetV
             // which is modeled here with: V + slope * (sum_branch qBranch) = TargetV - slope * qLoads + slope * qGenerators
-            equationSystem.createEquation(bus, AcEquationType.BUS_TARGET_V_WITH_SLOPE)
-                    .addTerm(vTerm)
+            equationSystem.getEquation(bus.getNum(), AcEquationType.BUS_TARGET_V).orElseThrow()
                     .addTerms(createReactiveTerms(bus, equationSystem.getVariableSet())
                             .stream()
                             .map(term -> term.multiply(slope))
                             .collect(Collectors.toList()));
-        } else {
```

Comment:
Where are these lines now? You only create the slope equation now.

Reply:
Already created